app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java

```diff
@@ -25,7 +25,6 @@
 import org.fossasia.openevent.app.data.event.Event;
 import org.fossasia.openevent.app.databinding.AboutEventFragmentBinding;
 import org.fossasia.openevent.app.ui.ViewUtils;
-
```

Comment:
Unneeded change. Kindly check the change list after updating PR to save time in review request cycles